src/AIAPI/MainGame/GameAPI.Hooks.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections;
+// using ActionGame;
+using HarmonyLib;
+using AIProject.SaveData;
+using Manager;
+using AIProject;
+using Sirenix.Serialization;
+
+namespace KKAPI.MainGame
+{
+    public static partial class GameAPI
+    {
+        private class Hooks
+        {
+            public static int lastCurrentDay = 1;//Always starts at 1
+
+            public static void SetupHooks()
+            {
+                Harmony.CreateAndPatchAll(typeof(Hooks));
+            }
+
+            //TODO SaveData.Load only has 1 param in AI and it's the path, does not include filename like KK
```

Comment:
Unlike KK, AI doesn't use path and filename together to save card data.  Its either one or the other.  Need to test this out.  You  can compare with KK/GameAPI.Hooks.cs to see what I mean